WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -457,7 +457,7 @@ private struct ProductsSection: View {
                         ProductDiscountView(imageURL: selectedProduct.productRowViewModel.imageURL,
                                             name: selectedProduct.productRowViewModel.name,
                                             stockLabel: selectedProduct.productRowViewModel.stockQuantityLabel,
-                                            productRowViewModel: selectedProduct.productRowViewModel,
+                                            productRowViewModel: productRow.productRow,
```

Comment:
Wow nice catch! I reverted the changes there in https://github.com/woocommerce/woocommerce-ios/pull/11383/commits/1fa814827288ac61000a3004d6c44ce885cea78e, where I updated to use `CollapsibleProductRowCardViewModel` just because `hasDiscount` was moved from `ProductRowViewModel` to  `CollapsibleProductRowCardViewModel`. As in a future subtask of https://github.com/woocommerce/woocommerce-ios/issues/11357, it'd be great to replace `ProductRowViewModel` with a new view model that's just for the discount view (especially `CollapsibleProductCardPriceSummary`).

Reply:
Btw noticed a pre-existing issue https://github.com/woocommerce/woocommerce-ios/issues/11393 in the discount form, it's pretty minor though.